validate.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import json
+import sys
+from glob import glob
+from urllib.request import urlopen
+
+from jsonschema import validate
+
+JSON_SCHEMA_URL = "https://raw.githubusercontent.com/openshift/insights-operator/f9b762149cd10ec98079e48b8a96fc02a2aca3c6/pkg/gatherers/conditional/gathering_rule.schema.json"
```

Comment:
Wouldn't be better to have the JSON schema here in this repository? It would mean that IO needs to read it from here (or from the image containing everything)

Reply:
Well, I think we can only duplicate it. Because schema is important and changing it should go through the regular review process as other changes to insights-operator.

Reply:
I am reluctant to duplicate it TBH (it's easier to have one source of truth). Yeah sure, but the regular review process can happen here or not? Comparing it to the IO review process with bugzillas, CI, backports, etc., I would say here it should be easier.

Reply:
Well, my point is that the schema is an important thing which allows the operator to not collect anything we don't intend (like data from non openshift namespaces), I'm not insisting on any of the solutions, but for openshift engineers it may be better when the schema goes through the same process as changing the operator's code.

Reply:
Well it's true that if we want to add a completely new condition then we will have to update the IO code anyway so let's leave it there now.